config/settings.d/puppet.yml.example

```diff
@@ -41,16 +41,20 @@
 # If you want to override the puppet_user above just for mco commands
 #:mcollective_user: peadmin
 
-# URL of the puppet master itself for API requests
+# Override use of Puppet's API to list environments, by default it will use only if
+# environmentpath is given in puppet.conf, else will look for environments in puppet.conf
+# (Puppet versions prior to 4.0 only)
+#:puppet_use_environment_api: true
+
+# Puppet master version used
```

Comment:
Even nitpickier: Puppet Server has its own version scheme, which contains a specific version of Puppet (like Puppet Agent w.x contains Puppet y.z).  I think this wording is slightly safer, or just remove "master".